docs/source/api.rst

```diff
@@ -38,6 +38,9 @@ Dataset Generation (Dask-based Multi-GPU)
   .. automodule:: cuml.dask.datasets.blobs
      :members:
 
+  .. automodule:: cuml.dask.datasets.regression
```

Comment:
I think we could also include the Single GPU and MNMG labelBinarizer in preprocessing.